lib/client/ca_export.go

```diff
@@ -71,7 +71,17 @@ func (r *ExportAuthoritiesRequest) shouldExportIntegration(ctx context.Context)
 	}
 }
 
-// ExportAuthorities returns the list of authorities in OpenSSH compatible formats as a string.
+// ExportedAuthority represents an exported authority certificate, as returned
+// by [ExportAllAuthorities] or [ExportAllAuthoritiesSecrets].
+type ExportedAuthority struct {
+	// Data is the output of the exported authority.
+	// May be an SSH authorized key, an SSH known hosts entry, a DER or a PEM,
+	// depending on the type of the exported authority.
+	Data []byte
+}
+
+// ExportAllAuthorities returns all authorities of a particular type in OpenSSH
```

Comment:
nit: should we update this description since other formats can be exported as well.

Reply:
Updated, well noted.